R/RestRserveApplication.R

```diff
@@ -106,7 +106,7 @@ RestRserveApplication = R6::R6Class(
       }
 
       # Generate UID
-      id = digest::digest(FUN)
+      id = digest::digest(FUN, algo = "murmur32")
```

Comment:
2 points here:
- there is non-zero probability of collision
- if we just want unique key we can use UUID

But overall I would prefer to avoid possibility of collission

Reply:
Agreed. Now hash calculated on the serialized function object. I don't know how get UUID for the raw or string.
Right now we can choose more strong hash algo.

Reply:
Also we can generate UUID in `add_route` and check existence with `any(sapply(self$handlers, identical, y = FUN))`.

Reply:
Why do you think it is necessary need to de-duplicate handlers? 

we can keep them in a list and avoid access with  hashing/uuid.

Reply:
I think it's rationally and should reduce memory usage when used many non unique handlers.
You are right we can keep them all.

Reply:
I push commit with UUID variant. See b48ed14.